src/neuroconv/datainterfaces/ophys/baseimagingextractorinterface.py

```diff
@@ -104,6 +104,7 @@ def add_to_nwbfile(
         metadata: Optional[dict] = None,
         photon_series_type: Literal["TwoPhotonSeries", "OnePhotonSeries"] = "TwoPhotonSeries",
         photon_series_index: int = 0,
+        nwbfile_module_name: Literal["acquisition", "processing"] = "acquisition",
```

Comment:
Looks like current agreement from other chain is `attach_to_nwbfile_module_name`